backstack/src/iOSMain/kotlin/com/slack/circuit/backstack/BackStackRecordLocalProvider.ios.kt

```diff
@@ -5,4 +5,4 @@ import androidx.compose.runtime.compositionLocalOf
 
 internal actual val LocalBackStackRecordLocalProviders:
   ProvidableCompositionLocal<List<BackStackRecordLocalProvider<BackStack.Record>>>
-  get() = compositionLocalOf { emptyList() }
+  get() = compositionLocalOf { listOf(SaveableStateRegistryBackStackRecordLocalProvider) }
```

Comment:
wait how did these ever work with the `get() = `???

Reply:
Great question, maybe they were only retrieved once? Or when they were an empty list, it didn't really matter if they were recreated. Fixed in latest commit.